service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/StandardDirectionOrderIssuedEmailContentProvider.java

```diff
@@ -35,4 +48,34 @@ public AllocatedJudgeTemplateForSDO buildNotificationParametersForAllocatedJudge
 
         return allocatedJudgeTemplate;
     }
+
+    public CTSCTemplateForSDO buildNotificationParametersForCTSC(CaseDetails caseDetails) {
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        HearingBooking hearing = caseData.getFirstHearing().orElseThrow(NoHearingBookingException::new);
+
+        CTSCTemplateForSDO ctscTemplateForSDO = new CTSCTemplateForSDO();
+        ctscTemplateForSDO.setRespondentLastName(getFirstRespondentLastName(caseData.getRespondents1()));
+        ctscTemplateForSDO.setCallout(buildCallout(caseData));
+        ctscTemplateForSDO.setCourtName(caseDataExtractionService.getCourtName(caseData.getCaseLocalAuthority()));
+        ctscTemplateForSDO.setHearingNeedsPresent(getHearingNeedsPresent(hearing));
+        ctscTemplateForSDO.setHearingNeeds(hearing.buildHearingNeedsList());
+        ctscTemplateForSDO.setHearingNeedsDetails(hearing.getHearingNeedsDetails());
+        ctscTemplateForSDO.setCaseUrl(getCaseUrl(caseDetails.getId()));
+        ctscTemplateForSDO.setDocumentLink(concatUrlAndMostRecentUploadedDocumentPath(xuiBaseUrl,
+            caseData.getStandardDirectionOrder().getOrderDoc().getBinaryUrl()));
+
+        return ctscTemplateForSDO;
+    }
+
+    private String buildCallout(final CaseData caseData) {
+        return "^" + buildSubjectLineWithHearingBookingDateSuffix(caseData.getFamilyManCaseNumber(),
+            caseData.getRespondents1(),
+            caseData.getFirstHearing().orElse(null));
+    }
```

Comment:
nice 👍 , Did we use to do this in template itself before? If i remember correctly, Do we need this same callout in many other places? If so, do you think we should extract it out somewhere!?